Fiction notes/Latex/note_CN_xlatex_template.tex

```diff
@@ -0,0 +1,1336 @@
+
+        if self.order == 4:
+            x = self.x0
+            y = self.y0
+            while x < 1:
+                y = runge_kutta_4(y, x, self.step, load_equ)
+                x += self.step
+                self.x_list.append(x)
+                self.y_sol.append(y)
+
+    def plot_result(self):
+        """Plots specific results"""
+        self.y_exact = [np.exp(self.params * x_i) + x_i for x_i in self.x_list]
+
+        def plot_se():
+            """Plots square error"""
+            se = ((np.array(self.y_exact) - np.array(self.y_sol)) ** 2)
+            plt.figure(num=1)
+            plt.plot(self.x_list, se, "k-", label="Square Error");
+            plt.legend();
+            plt.show();
+
+        def plot_solvedValue():
+            """Plots simulated value for each point"""
+            if self.order == 4:
+                table = pd.DataFrame(
+                    {
+                        "Selected_point": self.x_list,
+                        "Exact_Data_Point": self.y_exact,
+                        "Solve_Data_Point": self.y_sol
+                    }
+                )
+                print("The comparison table between exact value and " +
+                    "solved data point is printed here: ")
+                print(table)
+            else:
+                pass
+
+        def plot_solvedCurve():
+            """Plots solved curve"""
+            if self.order == 4:
+                plt.figure(num=2)
+                plt.plot(
+                    self.x_list,
+                    np.log10(np.array(self.y_exact)),
+                    "ro",
+                    MarkerFaceColor="w",
+                    MarkerEdgeColor="r",
+                    MarkerSize=5,
+                    label="Exact Solution, a=-50"
+                );
+                plt.plot(
+                    self.x_list,
+                    np.log10(np.array(self.y_sol)),
+                    "r-",
+                    label="Solved curve by 4 order R-K, a=-50"
+                );
+                plt.legend();
+                plt.ylabel("log10(y)")
+                plt.show();
+            else:
+                pass
+
+        plot_se()
+        plot_solvedValue()
+        plot_solvedCurve()
+
+
+case = RungeKutta()
+case.fit()
+case.plot_result()
+\end{python}
+
+%-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+%	SECTION:
+%-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+\section{结果与分析}
+
+\begin{figure}[!htbp]
+\centering
+\includegraphics[height=6.0cm,width=9.0cm]{5.1_1.png}
+\caption{不同参数下的精确解和计算结果}
+\label{2}
+\end{figure}
+
+\begin{definition}[(1) 参数讨论]\textit{ }\\
+
+如图2.1所示，分别赋予参数$a$一个小的正值，一个大的正值，一个绝对值小的负值和一个绝对值大的负值，可知在$a=50$时不能得到很好的收敛。四阶R-K方法稳定域可表达为$-2.785 \le \lambda h \le 0$，应用于本题得到参数$a$范围可表达为$-278.5 \le a \le 0$，结合题设条件得$-50 \le a \le 0$。因此，当参数$a$取超过0的数值，由于超出了4 阶R-K的稳定域结果收敛性相对较差。
+
+\end{definition}
+
+\begin{figure}[!htbp]
+\centering
+\subfloat[精确解和计算结果，a=-35, h=0.1]{\includegraphics[width=8.0cm]{5.1_2.png}}
```

Comment:
cause use package *subfig* rather than the package "subfigure" to avoid conflicts, here use "subfloat" to replace "subfigure"